README.md

```diff
@@ -1,165 +1,83 @@
-# NGINX Docker Image & Helm Chart for Openshift
+# nginx
```

Comment:
This changed the entire README.md, including valuable architecture and text. Find a way to keep them **and** adding the `values.yaml`